jujugui/static/gui/src/test/test_app.js

```diff
@@ -1512,12 +1473,12 @@ describe('App', function() {
       checkNext({root: 'store'});
     });
 
-    it('displays login if one of the apis is still connecting', () => {
```

Comment:
I'm pretty confident that it was an incorrect assumption based on the idea that you could be unable to log into a model the controller said you could - which isn't actually possible.